uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -483,6 +499,20 @@ define([
       };
 
       /**
+       * Checks if given status is applicable to manager
+       *
+       * @param {String} managerContactId
+       * @return {Boolean}
+       */
+      this.filterForManager = function (status) {
```

Comment:
generic name (filter for what?), use a more descriptive one, something like `isStatusAvailableForManager`